services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -206,29 +200,35 @@ qx.Class.define("osparc.data.model.NodeProgressSequence", {
       const clusterUpScalingPBar = this.__clusterUpScalingPBar = this.self().createProgressBar();
       this.__sequenceLoadingPage.add(clusterUpScalingPBar);
 
-      const pullingInputsTitle = this.__pullingInputsTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Pulling inputs..."));
-      this.__sequenceLoadingPage.add(pullingInputsTitle);
+      const pullingSidecarTitle = this.__pullingSidecarTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Setting up key components..."));
```

Comment:
I'd rephrase to: "Installing platform services..."

Reply:
I thought "Setting up key components...." is very vague. The user might wonder: are they setting up my s4l model, simulation, etc...?

While "Installing platform services...." suggests something they should not care about. 

General comment: in my opinion, users only care about their data, models, simulations, code, etc... and I think it's important that they understand that if they have big files for their models, simulations, etc, they need to be patient while we download them.

Reply:
My point was exactly that a too vague definition might induce users to think something else is happening (and something else is normally something they know). 😉

Reply:
Both are fine, with a gun to my head I prefer Elis suggestion